docs/src/use/configure/ignore.md

```diff
@@ -62,6 +62,61 @@ export default [
 ];
 ```
 
+## Global Ignores
+
+::: tip
+Global Ignores is what you need if you want to define files to ignore in a single place and avoid to add the same `ignores` prop to every configuration objects.
+:::
+
+In eslint, there is concept of `global ignores`, to better explain it let's see first how config object works in eslint.
+
+Eslint configuration file is intended to be used like this: export a single array, that contains multiple [configuration objects](https://eslint.org/docs/latest/use/configure/configuration-files#configuration-objects), each one being an item of the array.  
+
+It's important to note that each configuration object, it's independent from other ones. Because of that, the `ignores` prop defined in one configuration object is not applied to other configuration objects of the array.
+
+```js
+// eslint.config.js
+
+export default [
+    { name: "config-1", files: "src/*", ignores: [".dist/*"], ... }, 
+    { name: "config-2", files: "scripts/*", ignores: [".others/*"], ... }, 
+    // ...
+];
+```
+
+In the code above, the `ignores` directive of `config-1` is not applied on `config-2`, and vice-versa.  If this is what you want, use this pattern.
```

Comment:
Is true this part ? 

I'm not sure that what i wrote it's true, this is my personal assumption based on my usage of eslint, but I'm not an expert in eslint.
I want to be sure to not create more confusion...

Reply:
I'm referring specifically to 

> It's important to note that each configuration object, it's independent from other ones. Because of that, the `ignores` prop defined in one configuration object is not applied to other configuration objects of the array.


Configuration objects are independent?

Reply:
Yes they are.

Reply:
yes they are.